domain-models-runtime/src/main/java/org/folio/rest/persist/cql/CQLWrapper.java

```diff
@@ -76,9 +86,30 @@ public CQLWrapper setField(CQL2PgJSON field) {
     this.field = field;
     return this;
   }
+
   public String getQuery() {
-    return query;
+    if (whereClause != null) {
+      return whereClause;
+    }
+    if (criterion != null) {
+      return criterion.toString();
+    }
+    return query; // CQL query
+  }
+
+  public String getType() {
```

Comment:
Can you make this method private, or add javadoc explaining the possible return values?

Reply:
Sure. I had it public for a moment when logging queries in RestVerticle.